run.go

```diff
@@ -85,6 +86,18 @@ func Run(stepName string, signer cryptoutil.Signer, opts ...RunOption) (RunResul
 		return result, fmt.Errorf("failed to run attestors: %w", err)
 	}
 
+	fail := false
+	for _, r := range runCtx.CompletedAttestors() {
```

Comment:
We can use `errors.Join` to wrap multiple errors here. Could look something like

```go
errors := make([]error, 0)
for _, r := range runCtx.CompletedAttestor() {
    if r.Error != nil {
        errors = append(errors, r.Error)
    }
}

if len(errors) > 0 {
    return result, errors.Join(errors.New("attestors failed with errors"), errors...)
}
```